docs/src/main/asciidoc/spring-cloud-sleuth.adoc

```diff
@@ -841,7 +841,7 @@ Sleuth will search for beans of those types and automatically apply customizatio
 If a customization of client / server parsing of the HTTP related spans is required,
 just register a bean of type `brave.http.HttpClientParser` or
 `brave.http.HttpServerParser`. If client /server sampling is required, just
-register a bean of type `brave.http.HttpSampler` and name the bean
+register a bean of type `brave.sampler.SamplerFunction<HttpRequest>` and name the bean
```

Comment:
Isn't this a breaking change?

Reply:
I suppose you are right. because if a 3rd party is expecting the type `brave.http.HttpSampler`, then it doesn't matter that it implements `brave.sampler.SamplerFunction<HttpRequest>`. The scenario I'm
thinking is if someone is manually calling HttpTracing.builder?

One way out is to deprecate and rename ServerSampler -> HttpServerSampler
only check null on ServerSampler and use the old types as needed.

I can raise a change for that.